### ISSUE TITLE
Code Request

### ISSUE THREAD

**Issue by Owen864720655** [lrw890123/FactorVAE-pytorch#1]
Hi,

can you share the full code with the loss function definition, optimizer and the training results?

**Comment by lrw890123** [comment]
Sorry, I didn't reproduce successfully, so this is a half-finished product.



---- Replied Message ----
| From | ***@***.***> |
| Date | 03/01/2023 22:51 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [lrw890123/FactorVAE-pytorch] Code Request (Issue #1) |

Hi,

can you share the full code with the loss function definition, optimizer and the training results?
